Request/AbstractRequestBodyParamConverter.php

```diff
@@ -151,21 +163,55 @@ protected function execute(Request $request, ParamConverter $configuration)
     }
 
     /**
+     * @return Context
+     */
+    protected function getContext()
```

Comment:
I'm not sure that we should keep this method as ``Context`` is a final class...
``configureContext`` is there to customize the ``Context`` class.